src/js/smooch.jsx

```diff
@@ -146,11 +147,17 @@ export class Smooch extends Observable {
         }).then((loginResponse) => {
             store.dispatch(setUser(loginResponse.appUser));
             store.dispatch(updateReadTimestamp(getReadTimestamp()));
-            
+
             if (loginResponse.publicKeys) {
                 store.dispatch(setPublicKeys(loginResponse.publicKeys));
-            }
 
+                if (loginResponse.publicKeys.stripe) {
```

Comment:
The alternative would be to have an extra key in the response, `app.stripeEnabled = !!publicKeys.stripe`, which formalizes the assumption that public key means stripe enabled.